### ISSUE TITLE
Mobs should make sounds when attacking

### ISSUE THREAD

**Issue by kwadroke** [Wuzzy2/MineClone2-Bugs#94]
Most mobs do not make any sound when attacking or about to attack the player. 
I've been attacked from the back with no notice, just health dropping.
Examples: Creepers do not hiss before exploding unless being attacked. Slimes make sounds, but not when attacking. 

Currently only the spider makes sounds when attacking or about to attack the player.

**Comment by Wuzzy2** [comment]
Yeah, finding sounds for mobs is time-consuming and boring, but it needs to be done.
But if there is a Minecraft resource pack with sounds, released under a free culture/software license (e.g. MIT, BSD, WTFPL, CC BY, CC BY-SA, Unlicense, etc.), then it would be easy.

I am of course aware of this, but I will appreciate any help here. I mostly concentrate on the coding side for now. ;-)

Help includes: Pointing me to a usable Minecraft resource pack, or pointing to a single sound which I can use for a particular event. Possible sources are: Freesound.org, www.opengameart.org.

**Comment by kwadroke** [comment]
Mobs like Creepers have sounds, they just are rarely played especially before they blow up behind your back.
